checkstyle-tester/README.md

```diff
@@ -62,20 +65,27 @@ Please use `-Dcheckstyle.consoleOutput=true` option if you need to see all Check
 groovy launch.groovy projects-to-test-on.properties my_check.xml -Dcheckstyle.consoleOutput=true
 ```
 
-**Attention:** this project by deafult uses released version of Checkstyle and sevntu.checkstyle
-If you you need to use custom (snapshot) versions please update pom.xml to reference that versions ([checkstyle version](https://github.com/checkstyle/contribution/blob/master/checkstyle-tester/pom.xml#L29), [sevntu.checkstyle version](https://github.com/checkstyle/contribution/blob/master/checkstyle-tester/pom.xml#L34)), and please make sure that custom(newly generated) versions are located in your local maven repo 
+**Attention:** this project by default uses the latest SNAPSHOT version of [Checkstyle](https://github.com/checkstyle/contribution/search?utf8=%E2%9C%93&q=path%3Acheckstyle-tester+filename%3Apom.xml+%22checkstyle.version%22&type=) and the latest released version of [sevntu.checkstyle](https://github.com/checkstyle/contribution/search?utf8=%E2%9C%93&q=path%3Acheckstyle-tester+filename%3Apom.xml+%22sevntu.checkstyle.version%22&type=).
+If you you need to use custom (snapshot) versions please update pom.xml to reference that versions ([checkstyle version](https://github.com/checkstyle/contribution/blob/35d35dfcc48e2022403231e41aac8bf96126acc9/checkstyle-tester/pom.xml#L15), [sevntu.checkstyle version](https://github.com/checkstyle/contribution/blob/35d35dfcc48e2022403231e41aac8bf96126acc9/checkstyle-tester/pom.xml#L16)), and please make sure that custom(newly generated) versions are located in your local maven repo 
 
 ```
 ls  ~/.m2/repository/com/puppycrawl/tools/checkstyle/
 ls  ~/.m2/repository/com/github/sevntu/checkstyle/sevntu-checkstyle-maven-plugin
 ```
 
-to build SNAPSHOT version of `checkstyle` please run in his repo
+To build SNAPSHOT version of `checkstyle` please run in its folder (local git repository):
 
 ```
 mvn clean install -DskipTests -Dcobertura.skip=true -Dfindbugs.skip=true -Dpmd.skip=true
 ```
 
+**Attention:** 
+1) If you make changes to version X to change logic and pom.xml specifies version Y, you will not see any new/different violations from your changes. 
+   You will not get an exception message and unless you are expecting a specific difference, you may just assume your report is correct when it really isn't.
+   launch.groovy has no way to protect against this as branch with change isn't specified.
+2) Make sure that the versions of Checkstyle and SevNTU Checkstyle artifacts, which are located in your local Maven repository, corresponds to the versions specified in pom.xml,
+   otherwise there will be the exception, because maven-checkstyle-plugin will not find the required artifact in the local Maven repository.
```

Comment:
What I was saying in my previous point was shouldn't the bullets be like this:
````
Make sure that the versions of Checkstyle and SevNTU Checkstyle artifacts, which are located in your local Maven repository, corresponds to the versions specified in pom.xml,
1) otherwise there will be the exception, because maven-checkstyle-plugin will not find the required artifact in the local Maven repository.
2) You will not see any new/different violations from your changes, especially if you make changes to version X with changed logic and pom.xml specifies version Y. 
You will not get an exception message and unless you are expecting a specific difference, you may just assume your report is correct when it really isn't.
launch.groovy has no way to protect against this as branch with change isn't specified.
````

First sentence shouldn't be a point but the header to the 2 points as both points talk about version mis-match.